app/api/dao/user.py

```diff
@@ -200,7 +207,7 @@ def update_user_profile(user_id, data):
                 user.location = data['location']
             else:
                 user.location = None
-            
+
```

Comment:
Do we need this line break?

Reply:
That line break was already there. It just had 3 `tab`s, and I (or rather PyCharm) removed it so it conforms to the rest of the codebase.
But because we are told only to include necessary changes, I added those 3 `tab`s back.